mne/epochs.py

```diff
@@ -879,6 +879,10 @@ def drop_log_stats(self, ignore=['IGNORED']):
         -------
         perc : float
             Total percentage of epochs dropped.
+
+        See Also
+        --------
+        plot_drop_log
```

Comment:
I guess I didn't put the brief blurb where I thought it was obvious or when I was too lazy :P